### ISSUE TITLE
Translations

### ISSUE THREAD

**Issue by Th3R3alPr0** [pixelant/pxa_survey#14]
Is it possible to make backend and CSV-Export translations to german? Overwriting the locallang via TypoScript only works for frontend.

**Comment by anjeylink** [comment]
i'll take a look into this

**Comment by anjeylink** [comment]
fixed in #16 
it's possible now to add translations like below:
```
module.tx_pxasurvey {
_LOCAL_LANG.LANGUAGE_CODE_HERE.module\.see_analysis = Translation
}
```
